misaligned_fixed&Test.py

```diff
@@ -0,0 +1,31 @@
+
+def print_color_map():
+    major_colors = ["White", "Red", "Black", "Yellow", "Violet"]
+    minor_colors = ["Blue", "Orange", "Green", "Brown", "Slate"]
+    list = []
+
+    for i, major in enumerate(major_colors):
+        for j, minor in enumerate(minor_colors, 1):
+            interval = i * 5 + j

```

Comment:
I guess there has to be a `+ 1` here?

Reply:
I think that +1 is next to "enumerate(minor_colors, 1)" To start counting on 1 and not 0 (?)